package.json

```diff
@@ -986,6 +986,16 @@
               "onCommand:confluent.scaffold"
             ]
           },
+          {
+            "id": "create-connections",
+            "title": "Create a New Connection",
+            "description": "Click the + icon next to Resources -> Other to establish a connection to an Apache Kafka® Cluster or Schema Registry: in Confluent Cloud, Confluent Platform, or hosted elsewhere.",
+            "media": {
+              "image": "resources/walkthrough/direct-connection-form.png",
+              "altText": "Create a new connection to a Kafka cluster or Schema Registry"
+            },
+            "completionEvents": []
+          },
```

Comment:
```suggestion
          {
            "id": "create-connections",
            "title": "Connect Directly to Kafka / Schema Registry",
            "description": "In the Resources view, click the $(plus) icon next to the 'Other' item to connect directly to a Kafka cluster or Schema Registry instance.\n[Open Connection Form](command:confluent.connections.direct)",
            "media": {
              "image": "resources/walkthrough/direct-connection-form.png",
              "altText": "Create a new connection to a Kafka cluster or Schema Registry"
            },
            "completionEvents": [
              "onCommand:confluent.connections.direct"
            ]
          },
```
What do you think about something like this? Having the quick-access command buttons is handy, plus we can use the theme icon ids directly to match what we display in the sidebar views/items.
<img width="1113" alt="image" src="https://github.com/user-attachments/assets/c91d255a-e679-4d2e-9f2f-929d9a1e548a" />